src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -726,6 +726,60 @@ theorem getLsb_sshiftRight (x : BitVec w) (s i : Nat) :
         Nat.not_lt, decide_eq_true_eq]
       omega
 
+/-! ### signExtend -/
+
+/-- Equation theorem for 'Int.sub' when both arguments are 'negSucc' -/
```

Comment:
This documentation still mentions `negSucc`?

Also, what is an equation theorem?

Reply:
Resolved, thanks.